index.html

```diff
@@ -64,6 +62,39 @@ <h2>
     <p>
       TBW
     </p>
+    <p id="credential-type-examples">
+      There are many types of digital credential that can be conveyed using
+      this API. Some of these types include:
+    </p>
+    <ul>
+      <li>
+        a driver's license issued by a testing facility
+      </li>
+      <li>
+        a passport issued by a government
+      </li>
+      <li>
+        a student identification card or diploma issued by a university
+      </li>
+      <li>
+        an employee identification card issued by a company
+      </li>
+      <li>
+        a traveler's boarding pass issued by an airline
+      </li>
+      <li>
+        a profile issued by an online social network
+      </li>
+      <li>
+        a membership card issued by a retailer
+      </li>
+      <li>
+        a healthcare record issued by a hospital, and
+      </li>
+      <li>
+        a professional license issued by a trade association.
+      </li>
```

Comment:
Same with these two... these might end up being supported... but they are a bit murky right now.

Reply:
First one is same as comment in: https://github.com/WICG/digital-identities/pull/82/files#r1499358644

2nd item is a use case that is discussed often in the Verifiable Credentials for Education community. /cc @dmitrizagidulin @longpd @kayaelle

Generally speaking, I'm not suggesting we keep/remove any to any of these examples... wherever consensus lands is fine with me. I'm just noting that this text already existed and I didn't want to remove it entirely -- was meaning this PR to be a purely editorial PR.

Reply:
Right, I wouldn't read anything into "text was already there", as it's mostly just my initial ramblings.... which now reflected back at me 🤦

Reply:
Reduced the list down to the only credential type we have consensus on right now: 06271a72f5fcfbef716f61d49574b0de9449d5d0. Preserved the full list in https://github.com/WICG/digital-identities/issues/85#issuecomment-1963077832.